packages/components-react/calendar-react/src/insert-style.ts

```diff
@@ -0,0 +1,8 @@
+export const insertStyle = function (css: string) {
+  if (typeof document !== 'undefined') {
+    var head = document.head || document.getElementsByTagName('head')[0];
+    var style = document.createElement('style');
+    style.appendChild(document.createTextNode(css));
+    head.appendChild(style);
+  }
+};
```

Comment:
How about having this function as a utility under something like this @utrecht/utils.